packages/react/src/River/RiverBreakout.tsx

```diff
@@ -0,0 +1,68 @@
+import React, {Ref, forwardRef} from 'react'
+import clsx from 'clsx'
+import {RiverProps} from '.'
+import {useAnimation} from '../animation'
+import styles from './River.module.css'
+import {Heading} from '../Heading'
+
+type RiverBreakoutProps = {a11yHeading: string; a11yHeadingTag?: 'h2' | 'h3'} & Omit<
```

Comment:
Not sure if this is prettier not being active, but this should appear indented on newlines. If it's not automatically doing it, could we do that for consistency and easier scanning?

Reply:
Yea it looks like I'm having trouble with Prettier - even when I run it directly in the command line 🤔

Reply:
Looks like prettier is working then 🤔, weird. Do you know if it's using the workspace config? You can leave as-is I think, as the CI checks are passing now.

Reply:
I thought it was, but it didn't catch this one. Maybe my formatting actually _is_ correct even though it looks incorrect.